fe/fe-core/src/main/java/org/apache/doris/statistics/HiveAnalysisTask.java

```diff
@@ -85,7 +86,7 @@ protected void getColumnStatsByMeta() throws Exception {
         Map<String, String> parameters = table.getRemoteTable().getParameters();
         // Collect table level row count, null number and timestamp.
         setParameterData(parameters, params);
-        params.put("id", String.valueOf(tbl.getId()) + "-" + String.valueOf(col.getName()));
+        params.put("id", String.valueOf(tbl.getId()) + "--1-" + col.getName());
```

Comment:
I think we should extract a method to do this, currently, it is very error prone